homeassistant/components/xiaomi.py

```diff
@@ -107,7 +107,7 @@ def play_ringtone_service(call):
         else:
             ringtone = {'mid': ring_id, 'vol': int(ring_vol)}
 
-        gw_sid = call.data.get(ATTR_GW_SID)
+        gw_sid = call.data.get(ATTR_GW_SID).replace(":", "").lower()
```

Comment:
We should replace ATTR_GW_SID with ATTR_GW_MAC, since we are using mac in the config